scripts/check_doc_links.py

```diff
@@ -48,27 +48,29 @@
 IGNORE: Set[str] = {"https://faucet.metamask.io/", "https://ipfs.io/"}
 
 
-def is_url_reachable(url: str) -> bool:
+def is_url_reachable(url: str, attempts: int = 3) -> bool:
```

Comment:
I added retries here as this is flaky and again leading to unnecessary delay in passing tests

Reply:
We replace the checker script in open-autonomy with this one also.